variables.tf

```diff
@@ -59,37 +59,49 @@ variable "dead_letter_queue_destination" {
   description = "Send failed events/function executions to a dead letter queue arn sns or sqs"
 }
 
-variable "subscribed_log_group_names" {
-  description = "Log groups to subscribe to"
+variable "subscribed_s3_bucket_arns" {
+  description = "List of additional S3 bucket ARNs to subscribe lambda to."
   type        = list(string)
   default     = []
 }
 
+variable "subscribed_log_group_matches" {
+  description = <<-EOF
+    A list of regex patterns describing CloudWatch log groups to subscribe to.
 
-variable "subscribed_s3_bucket_arns" {
-  description = "List of additional S3 bucket ARNs to subscribe lambda to."
+    See https://github.com/observeinc/terraform-aws-cloudwatch-logs-subscription#input_log_group_matches for more info"
+  EOF
+  type        = list(string)
+  default     = [".*"]
```

Comment:
Might be a brave default for now.

My suggestion is to start this off as an empty list and retain backward compatibility with `subscribed_log_group_names`. We can update examples to include `".*"` and trial it on a few different target accounts.

Once we're confident that this works robustly across many different environments, we can flip the default and increment the major version of the module, providing upgrading instructions and removing `subscribed_log_group_names`.

Reply:
done